far/FarEng.hlf.m4

```diff
@@ -353,9 +353,9 @@ $ #Panel control commands#
  Deselect files with the same extension as the        #Ctrl+<Gray ->#
  current file
  Invert selection including folders                   #Ctrl+<Gray *>#
- (ignore command line state)
```

Comment:
I did not notice any interactions of any hotkeys in this group with command line state. Even `<Gray *>` apparently is not affected by the state of the command line. There is no reason to make this note.